### DISCUSSION TITLE
Adding a CSS Class for Active or Current Note

### DISCUSSION THREAD

**Discussion by bodiequirk** [SkepticMystic/breadcrumbs#487] [Ideas]
Hey Ross!

I wanted to know if there was a CSS class for the current or active note in Obsidian. I use Breadcrumbs as a dynamic hierarchy for my files and would love for the current note to have a different color than the other notes so I can glance over to the matrix and confirm which note is active due to its different color.

Is this possible? Thanks so much!! I love Breadcrumbs!!

**Comment by bodiequirk** [comment]
I would still love for this to be able to be done with Breadcrumbs! Can we do it yet? I thought this would work, but it does not.

.BC-active-note div {
	color: #ffffff !important;
}

Thanks!

**Comment by michaelpporter** [comment]
Already available: the current note gets the `.BC-active-note` class across the Tree, Matrix, nested edge list, and codeblocks, so you can target it in a CSS snippet. Documented on the docs site. Closing as resolved — thanks!
